app/Http/Controllers/CommunityController.php

```diff
@@ -40,8 +45,44 @@ public function getChat()
         return view('community.chat');
     }
 
-    public function getLive()
+    public function getLive(Request $request)
     {
-        return view('community.live');
+        $streams = null;
+        $featuredStream = null;
+        $streams = Cache::remember('livestreams', 5, function () {
+            $twitchApiUrl = config('osu.urls.twitch_livestreams_api');
+            $data = json_decode(file_get_contents($twitchApiUrl));
+
+            return $data->streams;
+        });
+        $featuredStreamId = Cache::get('featuredStream');
+        if ($featuredStreamId !== null) {
+            foreach ($streams as $stream) {
+                if ($stream->_id != $featuredStreamId) {
```

Comment:
I wasn't sure if twitch api returned it in form of string, but i checked it and you're right :( (fixed)